mlflow/sagemaker/__init__.py

```diff
@@ -205,6 +205,10 @@ def deploy(app_name, model_path, execution_role_arn=None, bucket=None, run_id=No
     Deploy an MLflow model on AWS SageMaker.
     The currently active AWS account must have correct permissions set up.
 
+    This function will create a SageMaker endpoint. For more information about the input data
+    formats accepted by this endpoint, see the
```

Comment:
This function will create a SageMaker endpoint.  -> This function creates a SageMaker endpoint.